iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
got an example of the should-use case ?

wish these emails had context... 8( see sawing between email the the browser

On 7/14/16, 3:42 PM, Glenn Matthews wrote:
>> @@ -124,10 +149,10 @@ def main(argv):
>>       # Handle Input ISO (Local or URI)
>>       if re.search(':/', args.ISO_FILE):
>>           # URI Image
>> -        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>> +        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>
> Either form is valid but `.format` is a bit more powerful.
>
> I learned recently that you *should* use %s-style for construction of logging messages because it permits lazy evaluation. For string construction, I tend to prefer .format but I don't have a strong opinion.
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ios-xr/iosxrv-x64-vbox/pull/10/files/fc29a814958e64ddff6a567916527f1c4084fe76#r70870010
>